apps/src/templates/alert.jsx

```diff
@@ -8,7 +8,8 @@ var Alert = React.createClass({
   propTypes: {
     children: React.PropTypes.element.isRequired,
     type: React.PropTypes.oneOf("error", "warning").isRequired,
-    onClose: React.PropTypes.func.isRequired
+    onClose: React.PropTypes.func.isRequired,
+    sideMargin: React.PropTypes.number,
```

Comment:
I prefer the `sideMargin` optional argument to an entire `style` object - esp. since the alert consists of at least two distinct elements, and it's a little ambiguous which one the style would apply to.  Also, I think a wide-open argument like that sometimes invites poor design decisions down the road.